### PR TITLE
[ADD] now lets the user rather approve or not comments on the products on the website, this in order to validate whether you want to display the comment or not.

### PR DESCRIPTION
**Pull request by oscarolar**
[ADD] now lets the user rather approve or not comments on the products on the website, this in order to validate whether you want to display the comment or not.

### CODE REVIEW

File: website_comment_approval/static/src/xml/mail.xml [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<template>
+    <t t-name="mail.thread.message">
```
Comment (nhomar) [inline website_comment_approval/static/src/xml/mail.xml]: Esto no se pudo lograr heredando?

**Comment by oscarolar** [comment]
Sorry, I left the file untouched, not ready yet...

**Comment by oscarolar** [comment]
@nhomar here is the [HU](https://www.vauxoo.com/?ts=1426631879796#id=365&view_type=form&model=user.story&menu_id=557&action=726) of this PR,please review and merge if possible.

**Comment by nhomar** [comment]
@oscarolar I merged in order to move forward this, but I have a question, I can't see in the code where you are adding the button itself which you click to approve, I could only see the server side method and the dataset instaciation but anything else, the jquery operation is not adding anything, can you tell me how was the technique iused to add the button which "approve" the message?

**Pull request merged**